config/crd/bases/hydra.ory.sh_oauth2clients.yaml

```diff
@@ -76,6 +76,14 @@ spec:
                     ClientName is the human-readable string name of the client
                     to be presented to the end-user during authorization.
                   type: string
+                deletionPolicy:
+                  description: |-
+                    Indicates if a deleted OAuth2Client custom resource should delete the database row or not.
+                    Value 0 means deletion of the OAuth2 client, value 1 means keep an orphan oauth2 client.
+                  enum:
+                    - 0
+                    - 1
```

Comment:
This seems to be wrong - 1 is delete and 2 is orphan. With this CRD, we cannot set `deletionPolicy: 2`, so it's impossible to use this new feature.

Reply:
Anyway, why were integers chosen here? This is extremely non-descriptive. Instead strings `Delete`/`Retain` should be supported.

Reply:
I reckon there is indeed a bug, feel free to open a pull request to address the issue :)